pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md

```diff
@@ -0,0 +1,147 @@
+    - Add the **Client scope** you created (`api`) to the assigned default client scopes.
+
+Finally, you'll need to add an user to be used by your API's client applications.
+
+## Configuring gatekeeper to proxy requests to your application
+
+**Keycloak-gatekeeper** will run as a sidecar inside your application **Pod**.
+By default all routes are protected. For more details check its
+[configuration options](https://www.keycloak.org/docs/latest/securing_apps/#configuration-options)
+
+This example here doesn't use helm for the sake of simplicity, but it should be easy to adapt to your specific scenario.
+
+### ConfigMap
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gatekeeper-config
+  namespace: mynamespace
+data:
+  keycloak-gatekeeper.conf: |+
+    # is the url for retrieve the OpenID configuration - normally the <server>/auth/realms/<realm_name>
+    discovery-url: https://login.syseleven.de/auth/realms/REALM_NAME
+    # skip tls verify
+    skip-openid-provider-tls-verify: true
+    # the client id for the 'client' application
+    client-id:  gatekeeper   # the secret associated to the 'client' application
+    client-secret: <CLIENT_SECRET>
+    # the interface definition you wish the proxy to listen, all interfaces is specified as ':<port>', unix sockets as unix://<REL_PATH>|</ABS PATH>
+    listen: :3000
+    # whether to enable refresh tokens
+    enable-refresh-tokens: true
+    secure-cookie: false
+    enable-logging: true
+    enable-default-deny: true
+    enable-session-cookies: true
+    no-redirects: true
+    ingress.enabled: true
+    debug: false
+    # the encryption key used to encode the session state
+    encryption-key: <random char sequence>
+    # the upstream endpoint which we should proxy request
+    upstream-url: http://127.0.0.1:8080/
+```
+
+### Deployment
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: myservice
+  namespace: mynamespace
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: myservice
+  template:
+    metadata:
+      labels:
+        app: myservice
+    spec:
+      containers:
+      - name: myservice
+        image: myservice:latest
+        ports:
+        - containerPort: 8080
+      - name: gatekeeper
+        image: keycloak/keycloak-gatekeeper:7.0.0
+        args:
+        - --config=/etc/keycloak-gatekeeper.conf
+        ports:
+        - containerPort: 3000
+          name: service
+        volumeMounts:
+        - name: gatekeeper-config
+          mountPath: /etc/keycloak-gatekeeper.conf
+          subPath: keycloak-gatekeeper.conf
+      volumes:
+      - name : gatekeeper-config
+        configMap:
+          name: gatekeeper-config
+```
+
+### Service
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: myservice
+  name: myservice
+  namespace: mynamespace
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: service
+  selector:
+    app: myservice
+  type: ClusterIP
+```
```

Comment:
can you add an example on how authentication then looks like / what the requests/responses are? e.g. some curl or some other simple app